web/pages/tour.html

```diff
@@ -5,7 +5,7 @@
     <p class="splashwelcome">
       This application allows you to interactively browse global satellite imagery within hours of it being acquired. Use the features described below to find interesting imagery, save and share what you find, and download the underlying data.
     </p>
-    <br>
+    <br><br>
```

Comment:
I'm not quite sure what caused the extra spacing in production, but I think a single space is fine, and fits the rhythm of the page better. A double `<br><br>` seems unnecessary. What do you think about removing it?

Reply:
I agree, I added it back in for parity with what we had before.